apps/token-manager/contracts/TokenManager.sol

```diff
@@ -40,7 +40,9 @@ contract TokenManager is ITokenController, IForwarder, AragonApp {
         bool revokable;
     }
 
-    mapping (address => TokenVesting[]) internal vestings;
+    // we are mimicing an array inside the outer mapping, we use a mapping instead to make app upgrade more graceful
+    mapping (address => mapping (uint256 => TokenVesting)) internal vestings;
+    mapping (address => uint256) internal vestingsLength;
```

Comment:
I'd rename this variable to `vestingLengths`, as we have multiple lengths in the mapping